android/app/src/main/AndroidManifest.xml

```diff
@@ -41,7 +41,7 @@
     android:usesCleartextTraffic="true"
     android:theme="@style/AppTheme">
     <meta-data android:name="com.bugsnag.android.API_KEY"
-               android:value="5aca2dbe708503471d8137625e092675"/>
+               android:value="BUGSNAG_API_KEY"/>
```

Comment:
Yikes. Good catch. Has the previous value been appropriately deprecated?

Reply:
No. Fortunately it's not much of a security issue.